src/resources/2020/Cartography.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Location,
+  Monster,
+  myTurncount,
+  runChoice,
+  runCombat,
+  toUrl,
+  useSkill,
+  visitUrl,
+} from "kolmafia";
+import { have as _have } from "../../lib";
+import { get } from "../../property";
+import { $skill } from "../../template-string";
+
+export const passive = $skill`Comprehensive Cartography`;
+export const skill = $skill`Map the Monsters`;
+
+export function have(): boolean {
+  return _have(passive);
+}
+
+export function mapMonster(location: Location, monster: Monster): void {
```

Comment:
We generally prefer returning a boolean to throwing; libram should only throw when it is catestrophic